src/err/mod.rs

```diff
@@ -477,7 +477,26 @@ impl PyErr {
     }
 
     /// Issues a warning message.
```

Comment:
Perhaps worth expanding this doc to say it's equivalent to `warnings.warn` (and similar for `warn_explicit`)?